pkg/helmexec/exec_test.go

```diff
@@ -289,7 +288,7 @@ exec: helm --kube-context dev upgrade --install release chart --timeout 10 --wai
 	buffer.Reset()
 	err = helm.SyncRelease(HelmContext{}, "release", "chart")
 	expected = `Upgrading release=release, chart=chart
-exec: helm --kube-context dev upgrade --install release chart
+exec: helm --kube-context dev upgrade --install release chart --history-max 0
```

Comment:
--history-max? why?

Reply:
pr: https://github.com/roboll/helmfile/pull/1022

Reply:
pr: https://github.com/roboll/helmfile/pull/1022

Reply:
got. but why 0?

Reply:
Probably because 0 is the empty value hence the default value for the HistoryMax field of ReleaseSpec? It's propagated via the --history-max flag here:
https://github.com/helmfile/helmfile/blob/1f134d931f3713088886803d045eb13760e4f731/pkg/helmexec/exec.go#L255
No behavior change introduced in this pull request. I presume it's just that we turned this helm v2 based test to helm v3 based one and the additional `--history-max` flag is the consequence of that.